machine_configuration/machine-config-node-disruption.adoc

```diff
@@ -24,13 +24,18 @@ The MCO does not validate whether a change can be successfully applied by your n
 
 For example, you can configure a node disruption policy so that sudo configurations do not require a node drain and reboot. Or, you can configure your cluster so that updates to `sshd` are applied with only a reload of that one service.
 
-:FeatureName: The node disruption policy feature
-include::snippets/technology-preview.adoc[]
-
 You can control the behavior of the MCO when making the changes to the following Ignition configuration objects:
 
 // I used this wording for the objects to match the previous section in the assembly: file:///home/mburke/openshift-docs/_preview/openshift-enterprise/mco-node-disruption-policy/post_installation_configuration/machine-configuration-tasks.html#what-can-you-change-with-machine-configs.
-* *configuration files*: You add to or update the files in the `/var` or `/etc` directory.
+* *configuration files*: You add to or update the files in the `/var` or `/etc` directory. You can configure a policy for a specific file anywhere in the directory or for a path to a specific directory. For a path, a change or addition to any file in that directory triggers the policy.
++
+[NOTE]
+====
+If a file is included in more than one policy, only the policy with the best match to that file is performed. 
```

Comment:
In this context, do we typically use the word _perform_ when referring to policies? That is, "the policy is performed"? That sounds unusual to me, but I'm not familiar with these docs. Would phrasing like this work? "...only the policy with the best match to that file is applied."